hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/ExpressionUtils.java

```diff
@@ -150,23 +150,27 @@ public static Object getValueFromLiteral(ValueLiteralExpression expr) {
     switch (logicalType.getTypeRoot()) {
       case TIMESTAMP_WITHOUT_TIME_ZONE:
         return expr.getValueAs(LocalDateTime.class)
-            .map(ldt -> ldt.toInstant(ZoneOffset.UTC).toEpochMilli())
-            .orElse(null);
+                   .map(ldt -> ldt.toInstant(ZoneOffset.UTC).toEpochMilli())
+                   .orElse(null);
       case TIME_WITHOUT_TIME_ZONE:
         return expr.getValueAs(LocalTime.class)
-            .map(lt -> lt.get(ChronoField.MILLI_OF_DAY))
-            .orElse(null);
+                   .map(lt -> lt.get(ChronoField.MILLI_OF_DAY))
+                   .orElse(null);
```

Comment:
Fix all the unnecessary changes for indentation.

Reply:
There is a problem with the submission format above, I follow the plugin 'Save Actions' configured by the community docs, as long as I save, this place will be formatted.

Reply:
Redundant format revert ,
But others will have problems modifying this file because it does not follow the check-style format。


@danny0405